src/context-providers/yup/types.ts

```diff
@@ -48,6 +49,7 @@ interface IYupRule {
 	excludes?: unknown | undefined;
 	uinfin?: boolean | undefined;
 	equalsField?: unknown | undefined;
+	withinDays?: IWithinDaysRule | undefined;
```

Comment:
`withinDays` should not be added to `IYupRule` because it will make this validation rule available to all fields but we just want it to be applied to `date-field`. (current changes to `date-field` work already)

we just want this rule to be available for conditional rendering, so it should be moved to `IYupConditionalValidationRule`.